compiler-core/src/docs.rs

```diff
@@ -196,6 +327,16 @@ fn function<'a>(
     }
 }
 
+fn text_documentation(doc: &Option<String>) -> String {
+    let raw_text = doc
+        .as_ref()
+        .map(|it| it.to_string())
+        .unwrap_or_else(|| "".to_string());
+
+    // TODO: parse markdown properly and extract the text nodes
```

Comment:
I think this MR should stop here and leave this TODO to the future.
As I said earlier, it will require adding a new dependency such as https://github.com/kivikakk/comrak and it will increase the scope of this MR too much.